src/components/ThemeSwitchMenu.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+
+import uuid from 'react-uuid';
+
+import {
+  MenuItem,
+  Select,
+  styled,
+} from '@mui/material';
+
+import { useAppDispatch } from '../app/hooks/useAppDispatch';
+import { selectTheme } from '../redux/selectedThemeSlice';
+import { themes } from '../themes/themes';
+import { store } from '../app/store';
+
+import { localStorageKeysObject } from '../utils/objects/localStorageKeys';
+
+const { 
+  themesKeys: {
+    theme,
+  },
+} = localStorageKeysObject;
+
+const localStorageThemeSubscriber = () => {
+  const state = store.getState();
+
+  localStorage.setItem(theme, state.theme.name);
+};
+
+const unsubscribe = store.subscribe(localStorageThemeSubscriber); // For later (maybe)
+
+const getMenuItemText = (selectedThemeName: string) => {
```

Comment:
```suggestion
const prepareThemeName = (selectedThemeName: string) => {
```

In this case better do not bound function name with the environment. And function name should reflect it's action (produced result)